### ISSUE TITLE
Block 11 - Preferred orientation?

### ISSUE THREAD

**Issue by ElanHR** [Statistical-Connectomics-Sp15/intro#98]
In the paper they characterize neurons by their preferred stimulus orientation but I'm a bit unclear what is being oriented. Does it just refer to which direction the axon/dendrites are aligned? 

http://en.wikipedia.org/wiki/Orientation_column#Preferred_Orientation

**Comment by aceeccc** [comment]
Yeah, it's referring to the direction the signals are being sent and the angle it's received in. Receptive fields of the neurons, or where stimuli will induce more neural firing, are orientation selective in some cases. So the neuron "prefers" and responds to stimuli of certain angles over others.

**Comment by edunnwe1** [comment]
Here's an example of preferred stimulus orientation: 
![image](https://cloud.githubusercontent.com/assets/10934557/6260770/a78361c4-b7b1-11e4-8cee-ba4cc7338364.png)
The basic experimental paradigm is that a bar is moving through the receptive field of the visual neuron, in this case probably a neuron in V1. 
On the left, you see the bar presented at different angles/orientations. On the right is the firing rate of the neuron in response to that stimulus (that oriented bar).
You can see that the preferred orientation of the neuron is vertical: the neuron prefers stimuli that are vertical. It still shows some activity to the bars that are near vertical because the tuning is not a delta function but smooth. 
So, this has nothing to do with the axon/dendritic alignment and everything to do with the firing rate of the neuron in response to a stimulus, and the category of stimuli that elicit the highest firing.

**Comment by yaxigeigei** [comment]
Following Erika's excellent explanation. If you are looking for some intuition about the underlying mechanism, you can conceptualize a neural network that taking input and spitting output. By changing the weights of synapses in certain way (or shaping them with training data), you can make the output specifically responding to certain bar orientation but not others. So, there are two important points. First, when we were just born and opened our eyes, our primary visual system had very strong and special unsupervised learning mechainsms and many neurons became classifiers for stimuli of particular feature. The underlying changes were mostly due to synaptic strenthening, weakening and pruning with limited change in the layout of axons and dendrites.

**Comment by yaxigeigei** [comment]
Following Erika's excellent explanation. If you are looking for some intuition about the underlying mechanism, you can conceptualize a neural network that taking input and spitting output. By changing the weights of synapses in certain way (or shaping them with training data), you can make the output specifically responding to certain bar orientation but not others. When we were just born and opened our eyes, our primary visual system had very strong and special unsupervised learning mechainsms and many neurons in V1 became classifiers for stimuli of particular feature (so do mice). The underlying changes were mostly due to synaptic strenthening, weakening and pruning with limited change in the layout of axons and dendrites.

**Comment by mblohr** [comment]
So preferred stimulus orientation is just a V1 neuron's response to visual stimulation of, say, bars in varying orientations?  And the bar orientation is noted as "preferred" when the neuron's response or firing rate is the highest?  Looks like the response distribution is normal on the circle.

**Comment by kristinmg** [comment]
Exactly. Most V1 neurons are orientation selective meaning that they respond strongly to bars of a particular orientation but not to the orthogonal orientation. Below is a similar figure Erica posted, but showing the distribution as well. The dashed rectantangles on the left indicate the neuron's receptive field and the superimposed lines are the line orientations that were used. For each orientation the number of action potentials produced by the neuron were recorded. The middle row are examples of EEG recordings for each of the corresponding stimulus orientations. The diagonal orientation (the middle row) evoked the greatest response. The graph on the right shows the results after a bunch of measurements. There is a peak response for one particular orientation and weaker responses for other orientations, falling of to zero when the line orientaiton is about 40 degrees away from the neurons preferred orientation.

![image](https://cloud.githubusercontent.com/assets/10712300/6312768/2591515e-b954-11e4-8afc-7f38a9555a90.png)

**Comment by mblohr** [comment]
Thank you - all of these figures are very helpful!
